### DISCUSSION TITLE
List associated interactions in the preparation grid & form

### DISCUSSION THREAD

**Discussion by grantfitzsimmons** [specify/specify7#5559] [Feature Requests]
Add the ability to list associated interactions in the preparation grid & form.

**Reported By**
UMich

**Comment by maxpatiiuk** [comment]
Should be possible on the `testability` branch thanks to the ability to change SubView from Form into Grid format.

Does that fix the issue?

**Comment by grantfitzsimmons** [comment]
That's not the issue that UMich was having. They want to see the interactions associated with each preparation in a list on the Preparation grid and table view.
